lib/service/service.go

```diff
@@ -1217,6 +1217,10 @@ func (process *TeleportProcess) initAuthService() error {
 		log.Errorf("PID: %v Failed to bind to address %v: %v, exiting.", os.Getpid(), cfg.Auth.SSHAddr.Addr, err)
 		return trace.Wrap(err)
 	}
+
+	authSSHAddr := cfg.Auth.SSHAddr
+	authSSHAddr.Addr = listener.Addr().String()
```

Comment:
what are some examples when `cfg.Auth.SSHAddr` and `listener.Addr().String()` are different?
leave a code comment please